src/templates/pluginVuex.ejs

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable no-await-in-loop */
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+import deepmerge from 'deepmerge';
+import extendsModules from '~/.nuxt/extends.modules';
+import storeModules from '~/.nuxt/store.modules';
+
+export default async ({ app }) => {
+    const storesForExtend = {};
+    const asyncForEach = async (array, callback) => {
+        for (let index = 0; index < array.length; index += 1) {
+            await callback(array[index], index, array);
+        }
+    };
+
+    await asyncForEach(Object.values(extendsModules), async (extendContent) => {
```

Comment:
Could you explain me what is the idea behind this? Even if Vuex store registering is a Promise - tho it's not. Using Promisses will not speed it up. For each step you are wrapping getting value at indeax as a callback and awaiting for it so it's like do step 1, wait for result and do step 2 and wait for result. Other thing would be to do Promise.all

Reply:
In this place, I have situation where `extendContent.extendStore[extendName]()` return Promise and I need waiting for this. So I used await loop, and close this mechanism in one place. Promise.all() need waiting for array and then I must looping for this information, so I choose await loop mechanism.